lib/basedriver/commands/session.js

```diff
@@ -89,7 +88,7 @@ commands.deleteSession = async function (/* sessionId */) {
   this.sessionId = null;
 };
 
-function fixCaps (originalCaps, desiredCapConstraints = {}) {
+function fixCaps (originalCaps, desiredCapConstraints = {}, logger = {}) {
```

Comment:
Having this default on `logger` will cause it to fail when used (since `{}` doesn't have a `warn` function on it) if there is no logger passed in. You'd need something like `, logger = {warn () {}}`